modes/config-mode/src/configMode.js

```diff
@@ -0,0 +1,184 @@
+import { hotkeys } from '@ohif/core';
```

Comment:
This is going to be heavily changed to allow it to create alternate modes and load them dynamically.

Reply:
It is now actually possible to load/create a new mode dynamically.  It is still quite heavily based on the existing longitudinal code, but working on that slowly.